pkg/collector/httpmetrics/json_path.go

```diff
@@ -26,21 +26,25 @@ func NewJSONPathMetricsGetter(httpClient *http.Client, aggregatorFunc Aggregator
 	return &JSONPathMetricsGetter{client: httpClient, aggregator: aggregatorFunc, jsonPath: compiledPath}
 }
 
-func DefaultMetricsHTTPClient() *http.Client {
+func CustomMetricsHTTPClient(request_timeout_ms uint64, connect_timeout_ms uint64) *http.Client {
```

Comment:
Please use `time.Duration` here for the arguments (related to my comment above).

Also Please use the Go style variable naming e.g. `camelCase` :) I would call it `requestTimeout`

Reply:
:+1: